Snippets/scale-font.py

```diff
@@ -0,0 +1,273 @@
+"""
+Script to change UnitsPerEm of a Font
+USAGE:
+scale-font.py path/to/inputFont.ttf targetUPM path/to/outputFont.ttf
+
+Only works on .ttf (no CFF) for now
+"""
+
+from fontTools import ttLib
+from sys import argv
+
+def scaleValue_toUPM(value, UPM, target_UPM):
+    if value < 0.0 :
+        return int ( ( value * target_UPM -0.5 ) / UPM )
+    else:
+        return int ( ( value * target_UPM +0.5 ) / UPM )
```

Comment:
please use `otRound` from `fontTools.misc.fixedTools` to round the scaled value. Import it at the top

Reply:
thanks!